src/include/session.h

```diff
@@ -62,6 +62,12 @@ struct __wt_session_impl {
     const char *lastop; /* Last operation */
     uint32_t id;        /* UID, offset in session array */
 
+    uint64_t operation_start_us;   /* Operation start */
+    uint64_t operation_timeout_us; /* Maximum operation period before rollback */
+#ifdef HAVE_DIAGNOSTIC
+    uint32_t op_5043_seconds; /* Temporary debugging to catch WT-5043, discard after 01/2020. */
```

Comment:
This shouldn't be needed.

Reply:
Removed the diagnostic code here and in `src/include/time.i`.